schedview/app/prenight_inventory.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import datetime
+import os
+
+import pandas as pd
+from rubin_sim.sim_archive import read_archived_sim_metadata
+
+from schedview.compute import munge_sim_archive_metadata
+from schedview.dayobs import DayObs
+
+SIM_TABLE_COLUMNS = (
+    "resource_url",
+    "sim_execution_date",
+    "sim_index",
+    "telescope",
+    "scheduler_version",
+    "opsim_config_version",
+    "opsim_config_script",
+    "tags",
+)
+
+
+def get_prenight_table(
+    day_obs: DayObs,
+    archive_uri="s3://rubin:rubin-scheduler-prenight/opsim/",
+    num_nights=7,
+) -> str:
+    """Create a data frame of pre-night simulations for a day obs.
+
+    Parameters
+    ----------
+    day_obs : `schedview.DayObs`
+        The date for which to get pre-night simulations.
+
+    Returns
+    -------
+    result : `pandas.DataFrame`
+        The table of prenight simulations.
+    """
+
+    # Make sure conditions are met.
+    if os.environ["LSST_DISABLE_BUCKET_VALIDATION"] != "1":
+        raise RuntimeError("Environment variable LSST_DISABLE_BUCKET_VALIDATION must be set to 1")
+    if "S3_ENDPOINT_URL" not in os.environ:
+        raise RuntimeError(
```

Comment:
Is this really an error or is it a warning? 
I thought the point of the archive_uri was that you could point it at anything, including a directory on disk.